Makefile

```diff
@@ -0,0 +1,133 @@
+# DO NOT EDIT BY HAND: is automatically re-generated by make.py
```

Comment:
I'm not sure about this.

We had auto-generated parts before, specifically `__init__.py`. If I remember correctly, we removed auto-generation in large part due to constant confusion over what needed to be updated.